doc/status-current-code.md

```diff
@@ -0,0 +1,63 @@
+## Current Network Code Overview
+
+The main responsabilities of Network configuration resides between yast2-network 
+and yast2 (NetworkInterfaces) and them are by sure the more complex parts of
+the code.
+
+- NetworkInterfaces (yast2): represent mainly the configured interfaces and is in
+  charge of writing the ifcfg-giles. The numbers of bugs in this area are not
+  so big, we have been dealing mostly with some normalization of the config or
+  better defaults like for wireless configuration.
```

Comment:
Since fixing device renaming, NetworkInterfaces is calm place ... however it needs some love. Some methods should be revised if it should live in an upper layer. Some features should be redesigned (filtering devices according to name when filtering to device groups)

Reply:
On the other hand this module will become less important in the future when we should support exporting device configuration in wicked's native xml.

Reply:
So in the end we will need a support for sysconfig way of configuration (NetworkInterfaces), wicked's xml, ... so we can even think of at least partly supporting NetworkManager